client/src/styles/trumps/icon.fonts.css

```diff
@@ -410,6 +410,10 @@ Generated using nucleoapp.com
   content: "\ea52";
 }
 
+.icon-test-apps::before {
+  content: "\ea55"
+}
+
```

Comment:
Sure, we can change the category and className test in AppIcon.test.js to visualization instead of test-apps. I'll take the .icon-test-apps css rule out on icon.fonts.css, as well